tesseract-server/src/handlers/util.rs

```diff
@@ -42,10 +42,6 @@ pub fn boxed_error_http_response(response: HttpResponse) -> FutureResponse<HttpR
     Box::new(future::result(Ok(response)))
 }
 
-
-pub const X_TESSERACT_API_KEY: &str = "x-tesseract-api-key";
```

Comment:
Since we're removing this, we can also remove line 92 from `tesseract-server/src/main.rs` as well as the `api_key` object from the `EnvVars` struct in `tesseract-server/src/app.rs`

Reply:
good catch! I'll remove those

Reply:
@MarcioPorto I've just pushed those removals